psa-ext/cert-meta.cddl

```diff
@@ -0,0 +1,16 @@
+$$endorsed-value-map-extension //= (
+  2 => psa-cert-meta
```

Comment:
we probably need to give `2` a symbolic name -- something like `corim.psa-cert-meta`